backend/prisma/schema.prisma

```diff
@@ -18,3 +31,44 @@ model User {
   failedLogins Int     @default(0)
   verified     Boolean @default(false)
 }
+
+model Category {
+  id        Int        @id @default(autoincrement())
+  name      String     @unique
```

Comment:
Is it possible for us to follow the naming conventions of the cleaned json as close as possible? So that when we reference back the cleaned the json, the table names make sense.

E.g Instead of name, lets call it a categoryTitle